apps/design/frontend/src/index.tsx

```diff
@@ -1,8 +1,19 @@
+import * as Sentry from '@sentry/browser';
 import React from 'react';
 import { createRoot } from 'react-dom/client';
 import { assert } from '@votingworks/basics';
 import { App } from './app';
 
+if (process.env.NODE_ENV === 'production') {
```

Comment:
@kofi-q Do you know if `process.env.NODE_ENV` will be available in client-side code? I saw that we use it in some other places, but couldn't find any configuration that made me sure

Reply:
I _believe_ it's set here for the frontend build: https://github.com/votingworks/vxsuite/blob/5cd82a14263f62d89753950ddfde1d2570445a96/Dockerfile.vxdesign#L70-L71

Reply:
+1 - `vite` should be passing that through to the bundle, so it's safe to use here